searchlib/src/vespa/searchlib/tensor/angular_distance.cpp

```diff
@@ -13,11 +13,11 @@ using vespalib::eval::Int8Float;
 
 namespace search::tensor {
 
-template<typename FloatType>
+template <typename FloatType, typename VectorStoreType>
```

Comment:
Given that cell type in distance function is always the same as the cell type in the VectorStore, if will be sufficient with just the VectorStoreType as a template argument.
Just add a public `using CellType = FloatType;` to each of Temporary and Reference store classes.

Reply:
Fixed